FV3/atmos_cubed_sphere/model/dyn_core.F90

```diff
@@ -1366,6 +1382,41 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_split, zvir, cp, akap,
           used=send_data(idiag%id_ws, ws, fv_time)
       endif
     endif
+    if (last_step .and. allocated(vulcan_omga)) then
```

Comment:
the parallel for this on omga only happens when using the hydrostatic, is this happening for the nonhydrostatic model intentional?

Reply:
Yes...it's a bit confusing.  The idea behind the "vulcan_omega" diagnostic is to compute omega in the same way it is computed in the hydrostatic model, but in the non-hydrostatic model.

Reply:
Oh like the descriptions says! hahah sorry :)